idea/resources/inspectionDescriptions/MainFunctionReturnUnit.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This inspection reports a main function should return <b>Unit</b> type.
```

Comment:
I'd rephrase this a little bit, like "a main function with incorrect return type (should be Unit)" or something like.